docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.2'
```

Comment:
Do we really need docker-compose here? We have only 1 container

Reply:
I added it for 3 reasons:
1. Configure easily the hot reloading, after every change in the code all containers be up-to-date automatically.
2. In the future we can add the backend & databases in the docker-compose to run all of them locally.
3. I was thinking in the possibility to add, in the future, multiple containers to run all networks at the same time in different ports:

```
version: '3.2'

services:
  safe-react-rinkeby:
    build:
      context: .
      dockerfile: ./docker/Dockerfile.Rinkeby
    volumes:
      - /app/node_modules
      - ./:/app
    ports:
      - 3000:3000

  safe-react-mainnet:
    build:
      context: .
      dockerfile: ./docker/Dockerfile.Mainnet
    volumes:
      - /app/node_modules
      - ./:/app
    ports:
      - 3001:3000

  safe-react-polygon:
    build:
      context: .
      dockerfile: ./docker/Dockerfile.Polygon
    volumes:
      - /app/node_modules
      - ./:/app
    ports:
      - 3002:3000

  safe-react-bsc:
    build:
      context: .
      dockerfile: ./docker/Dockerfile.BSC
    volumes:
      - /app/node_modules
      - ./:/app
    ports:
      - 3003:3000

``` 
So you can use the network that you want:
<img width="1339" alt="Captura de pantalla 2021-08-13 a las 14 37 28" src="https://user-images.githubusercontent.com/26763673/129358352-66eb3120-87e2-40fd-a618-7ea3171c1e1c.png">

Reply:
1. :+1: didn't know it was impossible without it
2. We did that before, for some reason it didn't work out and we stopped using it
3. I think one of the biggest goals for the future is to support multiple networks in a single app, because deploying a separate app with few changed variables is not optimal